src/datadog-service-catalog.ts

```diff
@@ -0,0 +1,214 @@
+import { JobPermission, JobStep } from 'projen/lib/github/workflows-model';
+import { NodeProject } from 'projen/lib/javascript';
+import { defaultServiceCatalogValues } from './utils/parameters';
+
+export module datadogServiceCatalog {
+  export interface ServiceInfo {
+    /**
+     * The name of the service. This must be unique across all services.
+     * @default project.name
+     */
+    readonly serviceName?: string;
+    /**
+     * Some details on what this service does
+     * @default 'Not Provided'
+     */
+    readonly description?: string;
+    /**
+     * The application/product that this service assists
+     * @default 'clickup'
+     */
```

Comment:
```suggestion
    /**
     * The next larger scope of functionality that this service helps to provide.
     * At ClickUp this is often a "Major Feature", but could be an even smaller scope for micro-services.
     * The goal here is to make it easier for people troubleshooting to find the relevant services involved.
     * @default 'Not Provided'
     */
```

`clickup` is a bad default. It's bad because
- it doesn't add any value (it doesn't help us understand what the service actually does)
- it excuses developers from telling us what the service does
- it's not relevant or appropriate for other non-clickup users of this open-source repo

I'd be ok with the last thing if we were getting value out of it, but...